integration/grafana/README.md

```diff
@@ -1,25 +1,147 @@
+- Assist with the initial one-time configuration of the Grafana server
+- Monitor jobs submitted to a PBS batch system
+- Review data collected from previously monitored jobs
+
+The script runs the Prometheus and Grafana server on the head node of a
+system. Optionally, the script may also deploy and connect to `geopmexporter(1)`
+clients running on the allocated nodes of a user's job.
+
+
+### Requirements
+
+A user installation of Grafana and Prometheus is required.  We recommend
+following the [Linux installation
+instructions](https://grafana.com/grafana/download?platform=linux) to `wget` the
+Grafana archive for Linux.  The expanded archive is the `GRAFANA_DIR` provided
+to the `pbs_prometheus_launch.py` script CLI.  To install Prometheus download
+the [archive for the latest stable Linux](https://prometheus.io/download/).  The
+expanded archive is the `PROMETHEUS_DIR` provided to the
+`pbs_prometheus_launch.py` script CLI.
+
+Until the `geopmexporter(1)` is available in a stable release, the user must
+build and install a development snapshot of GEOPM.
+
+```bash
+    # Install latest development snapshot of GEOPM
+    GEOPM_PREFIX=... # target installation directory
+    wget https://raw.githubusercontent.com/geopm/geopm/refs/heads/dev/geopmdpy/install_user.sh
+    chmod a+x install_user.sh
+    ./install_user.sh --prefix=$GEOPM_PREFIX --enable-levelzero
+```
+
+
+### Configuring ports
+
+There are three ports used by the script to communicate:
+
+- Prometheus server port opened on the head node (default 9090)
+- Grafana server port opened on the head node (default 3000)
+- Prometheus client port opened on the allocated nodes (default 8000)
+
+To avoid conflicts, or firewall limitations, the user may override the defaults
+using command line arguments to pbs_prometheus_launch.py
+
+### Configuring Grafana Server
+
+Before deploying the GEOPM Prometheus exporter across the system for the first
+time, the Grafana server needs to be configured on the head node by setting the
+administrator password and uploading the GEOPM Power Dashboard though the web
+GUI.
+
+To set the administrator password for your Grafana instance with the `grafana-cli`
+tool run the following command inside of the Grafana software directory:
+
+```bash
+    cd $GRAFANA_DIR/
+     ./bin/grafana-cli admin reset-admin-password $GRAFANA_ADMIN_PASSWORD
+```
+
+To add the GEOPM Prometheus dashboard to your Grafana configuration the Grafana
+and Prometheus servers must be running.  This is done by executing the
+`pbs_prometheus_launch.py` script without specifying the `--jobid` option.  This
+will bring up the Prometheus server and Grafana server on the head node.
+
+Then navigate to the Grafana web GUI in a web browser to import the GEOPM
+Dashboard.  The Grafana server will be running an http (not https) server on the
+head node URL on port 3000 (port can be modified with the the --graf-port option
+to `pbs_prometheus_launch.py`).  For example if the head node hostname is
```

Comment:
```suggestion
head node URL on port 3000 (or the port specified to `pbs_prometheus_launch.py`
via the `--graf-port` option). For example, if the head node hostname is
```

...or, as worded in the documentation below for adding the Prometheus data source